src/PluginsSystem/Microsoft.Performance.Toolkit.Plugins.Runtime/Validation/PluginsSystemOperation.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Performance.Toolkit.Plugins.Runtime.Validation
+{
+    /// <summary>
+    ///     Represents an operation within the plugins system that requires plugin validation.
+    /// </summary>
+    public enum PluginsSystemOperation
+    {
+        /// <summary>
+        ///     One or more plugins are attempting to be installed.
```

Comment:
>One or more plugins are attempting to be installed. [](http://example.com/codeflow?start=16&length=51)

nit: sounds like the plugin is doing the work.

Another comment in another file has this:

"The <see cref="IPluginValidator"/> for validating a plugin that is being installed."

I think we might want to change this, and the description below, to be more consistent. Maybe something like:

Attempting to install one or more plugins.